.storybook/themes/commerce.scss

```diff
@@ -202,6 +188,25 @@
 		"border-radius-pill": var(--global-border-radius-2),
 		"border-radius-circle": var(--global-border-radius-3),
 		// Typography
+		// headings
+		"heading-level-1-font-size": var(--global-font-size-12),
+		"heading-level-1-line-height": var(--global-line-height-3),
+		"heading-level-1-font-weight": var(--global-font-weight-7),
+		"heading-level-2-font-size": var(--global-font-size-11),
+		"heading-level-2-line-height": var(--global-line-height-4),
+		"heading-level-2-font-weight": var(--global-font-weight-7),
+		"heading-level-3-font-size": var(--global-font-size-10),
+		"heading-level-3-line-height": var(--global-line-height-4),
+		"heading-level-3-font-weight": var(--global-font-weight-7),
+		"heading-level-4-font-size": var(--global-font-size-9),
+		"heading-level-4-line-height": var(--global-line-height-5),
+		"heading-level-4-font-weight": var(--global-font-weight-7),
+		"heading-level-5-font-size": var(--global-font-size-7),
+		"heading-level-5-line-height": var(--global-line-height-5),
+		"heading-level-5-font-weight": var(--global-font-weight-7),
+		"heading-level-6-font-size": var(--global-font-size-5),
+		"heading-level-6-line-height": var(--global-line-height-5),
+		"heading-level-6-font-weight": var(--global-font-weight-7),
```

Comment:
Hey @nschubach started to review, but I think some line heights were missed for body, small and tiny on lines 213, 217 and 221. Mind taking a look?

Reply:
I updated it and "reduced" the amount of changes to 402 ...